src/CommunityToolkit.Maui.UnitTests/Views/GravatarImageSource/GravatarImageSourceServiceTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using CommunityToolkit.Maui.Core;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Views.GravatarImageSource;
+
+public class GravatarImageSourceServiceTests : BaseTest
```

Comment:
I am not sure we need these tests, at least the first will go if we do remove `IGravatarImageSource` but I would love to have some tests that do verify our concrete class `GravatarImageSource` continues to play with `UriImageSourceService` given the latter is outside of our control. I'd be happy to provide bits here if you need?

Reply:
@bijington If you have some GIST/repo I can look at, that would be great.